.github/labeler.yml

```diff
@@ -0,0 +1,30 @@
+":robot: CI":
+  - .github/**/*
+
+":book: docs":
+  - assets/**/*
+  - docs/**/*
+
```

Comment:
There is another label called `documentation` defined in `labels.json`. Shouldn't we settle for one, or do we want to distinguish between automatic labels and standard ones?